networkit/cpp/io/NetworkitBinaryGraph.md

```diff
@@ -20,25 +20,33 @@ struct Header {
     uint64_t offsetTranspose;
     uint64_t offsetWeightLists;
     uint64_t offsetWeightTranspose;
+    uint64_t offsetAdjIdLists;
+    uint64_t offsetAdjIdTranspose;
 };
 ```
 - magic: A constant value used to identify the file format version.
     - The current version is '*nkbg002*' which supports weighted, undirected and directed graphs.
 - checksum: Currently not used
 - features: Contains the graph information bitwise
     - Bit 0 : directed or undirected
-    - Bit 1-2 : the weight format of the graph. Below are the possible formats:
-        - 0 = Weights are unsigned integers
-        - 1 = Weights are signed integers
-        - 2 = Weights are doubles
-        - 3 = Weights are floats
+    - Bit 1-3 : the weight format of the graph. Below are the possible formats:
+		- 0 = Graph is unweighted
+        - 1 = Weights are unsigned integers
+        - 2 = Weights are signed integers
+        - 3 = Weights are doubles
+        - 4 = Weights are floats
+	- Bit 4 = proper edge indexing
```

Comment:
"proper edge indexing" -> "edge indices are present". Remove the unrelated changes above this one.

Reply:
This still needs to be fixed.